src/dataimport/TPDI.ts

```diff
@@ -222,8 +229,13 @@ export class TPDI {
       const requestConfig: AxiosRequestConfig = createRequestConfig(innerReqConfig);
       requestConfig.responseType = 'blob';
 
+      const params = new URLSearchParams();
```

Comment:
It would probably be better to move  `getThumbnail` to *DataProvider classes. fftr